### ISSUE TITLE
Differences from Plenoxels

### ISSUE THREAD

**Issue by paulmelis** [akanimax/relu_fields#15]
Sorry for asking something not directly related to the code. But ReLu fields seem to have much in common with Plenoxels (as noted in your paper), expect perhaps for the grid pruning used during coarse-to-fine training, and regularization step. But perhaps that matches with the focus mentioned in the paper "as to what is actually required for high-fidelity scene reconstruction". Given the similarities, though, I was wondering if you did compare PSNR values against Plenoxes?

**Comment by akanimax** [comment]
Hi, no worries. 
Indeed, ReLUFields share a lot in common with Plenoxels, but being a concurrent work, formal comparisons weren't included the paper. However, with the (not-so-formal) experiments I did, Plenoxels seem to have 2-3 dB higher PSNR on all scenes, because in their default setting, they use a spatially-adaptive grid (data-structure). And, also at a much higher resolution than the experiments we did (I think 512^3).
